SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -380,14 +348,28 @@ bool USpatialActorChannel::ReplicateActor()
 	}
 	else
 	{
-		for (UActorComponent* ActorComp : Actor->GetReplicatedComponents())
+		FOutBunch DummyOutBunch;
+
+		FClassInfo* ClassInfo = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
+
+		for (UActorComponent* ActorComponent : Actor->GetReplicatedComponents())
 		{
-			if (ActorComp && ActorComp->GetIsReplicated()) // Only replicated subobjects with type bindings
+			if (ClassInfo->SubobjectClasses.Contains(ActorComponent->GetClass()))
 			{
-				if(Interop->GetTypeBindingByClass(ActorComp->GetClass()))
-				{
-					bWroteSomethingImportant |= ReplicateSubobject(ActorComp, RepFlags);
-				}
+				bWroteSomethingImportant |= ReplicateSubobject(ActorComponent, RepFlags);
+				bWroteSomethingImportant |= ActorComponent->ReplicateSubobjects(this, &DummyOutBunch, &RepFlags);
+			}
+		}
+
+		for (UObject* Subobject : NetDriver->TypebindingManager->GetHandoverSubobjects(Actor))
+		{
+			// Handover shadow data should already exist for this object. If it doesn't, it must have
+			// started replicating after SetChannelActor was called on the owning actor.
+			TArray<uint8>& SubobjectHandoverShadowData = HandoverShadowDataMap.FindChecked(Subobject).Get();
+			FHandoverChangeState SubobjectHandoverChangeState = GetHandoverChangeList(SubobjectHandoverShadowData, Subobject);
+			if (SubobjectHandoverChangeState.Num() > 0)
+			{
+				Sender->SendComponentUpdates(Subobject, this, nullptr, &SubobjectHandoverChangeState);
 			}
 		}
 	}
```

Comment:
There is commented code on line 377 below, do we intend to keep it or can we remove it? do we need a Jira?